app/state/reducers/ui/reservationsReducer.js

```diff
@@ -29,6 +29,15 @@ function selectReservationToEdit(state, action) {
   });
 }
 
+function parseError(error) {
+  if (error.response && error.response.non_field_errors && error.response.non_field_errors.length) {
+    return error.response.non_field_errors.join('. ').replace('[\'', '').replace('\']', '');
+  } else if (error.response && error.response.detail) {
+    return error.response.detail;
+  }
+  return 'Jotain meni vikaan';
```

Comment:
This message is not translated but I guess it is fine, as the backend messages are in finnish too?